### DISCUSSION TITLE
Focus on Shadow Purify

### DISCUSSION THREAD

**Discussion by Antoninqlc** [Farigh/pokeclicker-automation#348] [Q&A] [answered]
I have already captured all the shadow pokemon and I just need to purify them, what is the purpose of the route the automatisation make me go ? It did'nt farm any achievement

**Comment by Farigh** [comment]
Once all shadow Pokemon are purified, the feature should disable itself, if it's not the case, it means you still have some to purify

It might be trying to get more pokedolars, or more dungeon token to continue farming those

**Comment by Farigh** [comment, marked as answer]
Ho sorry, you mean you caught them all and some are still not purified....

The automation will then try to purify those as fast as possible, usually by going to the highest "kill-rate" route to fill the shine

**Reply by Antoninqlc** [reply]
OK thanks, I don't know if it's even possible but an option to farm other things like pokerus cure or achievement while purify pokemon when it's possible could be pretty usefull

**Reply by Farigh** [reply]
I'll try to see if I can find a way, in the current state of the automation, it will not be easy to do.

Parallelize things from the same "Feature" is not possible
